docs/design/ovs-pipeline.md

```diff
@@ -749,10 +749,12 @@ connections - as a reminder all connections are committed in
 [ConntrackCommitTable] - packets go straight to IngressMetricsTable,
 then [L2ForwardingOutTable], with no other match required.
 
-Flow 2 ensures that traffic from the local gateway cannot be dropped because of
-Network Policies. This ensures that K8s [liveness
+Flow 2 ensures that the traffic initiated from the host network namespace cannot
```

Comment:
Do you like to mention NodePort traffic will not be skipped?

Reply:
Please check if the latest version makes sense to you.